### ISSUE TITLE
case_when stuff

### ISSUE THREAD

**Issue by reblake** [SESYNC-ci/advanced-tidyverse-lesson#16]
Need to include (RHS) and (LHS) abbreviations after your written out text to be clearer for when you use the abbreviations below.  

Also, case_when() can be tricky to figure out because the order of your things matters a lot.  IIRC, you have to go from most specific -> most general statements.  I think you need to go into the explanation a bit more here.

**Comment by qdread** [comment]
Yeah the specific to general thing is true. However if you have complicated logic, it can be hard to figure out which things are more specific than others. So to me a recommended best practice is to split up the logic more rather than trying to shoehorn it all into one ungodly long case_when statement.

**Comment by qdread** [comment]
Yes, after looking at the lesson I wasn't sure whether the example you provide is the best one to demonstrate use cases where `case_when` is really the most useful. To me, `case_when` is nice for doing things like labeling different classes of a continuous variable like

```
case_when(
  x < 10 ~ "small",
  x >= 10 & x < 20 ~ "medium",
  x >= 20 ~ "large")
```

but your example could be done by other means (maybe more intuitively), such as joining a lookup table of colors to the data or passing a named color vector to the color scale.

**Comment by khondula** [comment]
edited but can be better to have an example thats more integrated with the rest of the lesson, eg. making a new categorical value that is then used in forcats part and plot ordering
